### DISCUSSION TITLE
Need to change the file name format

### DISCUSSION THREAD

**Discussion by garysolebc** [CMSgov/price-transparency-guide#259] [Q&A] [answered]
The current file name format is <YYYY-MM-DD>_<payer or issuer name>_<plan name>_<file type name>.<file extension>, example 2020-01-05_cms_medicare_in-network-rates.json
However, with the recent change to have the plan name be in an array since so many plans share the same network, it seems that the file name format needs to change.
I suggest that the name relate more to the payer or issuer network name that defines the network the providers in the file belong too and the various plans/products share.

**Comment by bryanbriegel** [comment]
agree, the file naming convention needs to be update to support the change in the schema, and this is the level of change that would support the schema.  This question/ suggestion has been raised to CMS, and it would be helpful if this is addressed on the upcoming webinar.

**Comment by Devyandu** [comment]
Agreed. It makes sense for the file name to be a the payer level. Hopefully we get some guidance in the upcoming webinar.
I posted a [proposal ](https://github.com/CMSgov/price-transparency-guide/discussions/260)similar to your original post.

**Comment by shaselton-usds** [comment, marked as answer]
Putting a bow on this -- updates were made to support the [naming formats](https://github.com/CMSgov/price-transparency-guide#file-naming-convention).
